packages/frontend-core/src/api/user.js

```diff
@@ -156,13 +156,14 @@ export const buildUserEndpoints = API => ({
     return await API.post({
       url: "/api/global/users/onboard",
       body: payload.map(invite => {
-        const { email, admin, builder, apps } = invite
+        const { email, admin, builder, apps, appBuilders } = invite
         return {
           email,
           userInfo: {
             admin: admin ? { global: true } : undefined,
             builder: builder ? { global: true } : undefined,
             apps: apps ? apps : undefined,
+            appBuilders: appBuilders ? appBuilders : undefined,
```

Comment:
NAB: doing a conditional with `undefined` won't ever change anything (other than technically turning explicit `null` or `false` into `undefined`, but I doubt that's needed). So this can just be `appBuilders,`.